fix: image position on "Confirm" view

mdibuhossain opened the pull request:
fixes # [(88)](https://github.com/Azordev/maps-project/issues/88)

#### Screenshots

case 1 (fixed): Desktop > 1326px. The image is hiding in the bottom of the page
![image](https://user-images.githubusercontent.com/59278819/158653740-06d7201e-ed65-47ed-b833-e2e9c3aa6a12.png)

case 2 (fixed): Tablet > 600px AND < 946px. The image is overflow the container
![image](https://user-images.githubusercontent.com/59278819/158653960-7db927ee-9a96-43e0-943b-0dc74990bd3e.png)

case 3 (fixed): Tablet > 972px AND < 1024px The image is so big, please check how to fix it.
![image](https://user-images.githubusercontent.com/59278819/158654390-f04dba3a-e5f4-430e-ba0a-432d0ce3d72f.png)

### Checklist:

- [x] Please, fix all cases that I've left in the screenshots.

heliomar-pena commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
   width: auto;
-  margin: 0 30px;
+  margin-top: 10px;
+  // margin: 0 30px;
 
   @media (max-width: 960px) {
     position: relative;
     height: 100px;
-    bottom: -20px;
-    width: 94px;
+    // bottom: -20px;
+    width: 80px;
+    height: auto;
   }
```
Please avoid to mix `max width` with `min width` media queries.

You can try to use:

```css
  width: 80px;
  margin-top: 10px;
  height: auto;
  position: relative;
```

instead of use:

```css
 width: auto;
  margin-top: 10px;
  // margin: 0 30px;
  @media (max-width: 960px) {
    position: relative;
    height: 100px;
    // bottom: -20px;
    width: 80px;
    height: auto;
  }
```

Here we've:

1. Removed unused code (width auto, height 100px and comments code)
2. Removed unnecessary media max-width block code

heliomar-pena commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
   }
 
   @media (min-width: 961px) {
-    width: 20vw;
+    width: 80px;
     height: auto;
     bottom: 0;
     right: 50vw;
   }
```
Currently this `bottom` and `right: 50vw` is not doing anything since the element position is "static". Firefox says to me that it's not being applied to the element. Then I think can be removed

![imagen](https://user-images.githubusercontent.com/66505715/158848006-63b3e766-98b6-4f71-aa9e-3fc108f0e196.png)

If you want to know, the position is static because position relative is inner of `media query max-width 960px`, then it's not applying in this size. But we already have seen that this size looks good without position relative, bottom and right position, then please considerate if you want to add position relative or else please remove bottom and right property, since they're applying to the element

heliomar-pena commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
+    // margin-left: auto;
+    position: absolute;
+    left: 0;
+    bottom: 25%;
```
Good job! Please try to put the image to the right size in desktop.

Instead of:
![imagen](https://user-images.githubusercontent.com/66505715/158855511-dd060b68-be3f-4fa2-b142-3fad2d7f11e8.png)

Should looks some like: 
![imagen](https://user-images.githubusercontent.com/66505715/158855822-4d9e3e4c-bd2e-40b3-9acd-0226c457c01b.png)

I meant: Should be more near of the bottom of page, and should be at the right

heliomar-pena commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
-    position: relative;
-    left: 359px;
-    top: 352px;
-    width: 147px;
+    // margin-left: auto;
+    // position: relative;
+    // left: 359px;
+    // top: 352px;
+    // width: 147px;
   }
```
Please remove this block if it's unused

e-azocar replied:
If you're not using any code line in the `@media`, please remove it

heliomar-pena commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
 export const HeaderContainer = styled.div`
+  position: relative;
   margin: 5vw;
+  // background: red;
+  // border: 1px solid red;
```
Please remove comments:

```suggestion
```

Katsu08 reviewed (changes requested):
Good work man! I left you some comments

e-azocar commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
   width: auto;
-  margin: 0 30px;
+  margin-top: 10px;
+  // margin: 0 30px;
```
Please remove comments

e-azocar commented on src/pages/Confirm/Confirm.styled.jsx:
```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
     height: 100px;
-    bottom: -20px;
-    width: 94px;
+    // bottom: -20px;
```
Please remove comments

mdibuhossain commented:
@Katsu08 and @e-azocar thank you for instructing me.
I have changed the code and removed the comments as you required.
I pushed the code.
Should I go for the pull request again?

Katsu08 reviewed (approved):
Good work man !

e-azocar reviewed (approved):
Really good job

e-azocar commented:
@mdibuhossain No, when you push your new code in the same branch of the Pull Request, the changes updated automatically

Katsu08 reviewed (approved):
Good job!
